openvox/install.md

```diff
@@ -57,7 +57,23 @@ Then install the packages you want.
 
 If you have backed up config files, then restore them now.
 
-### Sponsorship
+### Managing OpenVox with OpenVox
+
+You can manage OpenVox with several existing modules:
+
+* [`puppet-puppet` from `theforeman`](https://github.com/theforeman/puppet-puppet)
+* [`puppetlabs-puppetdb`](https://github.com/puppetlabs/puppetlabs-puppetdb)
+
+To prevent the modules from trying to install Puppet, you can use this `hiera` data:
```

Comment:
Maybe update the wording a bit? When you set those parameters on an existing Perforce installation (and have the openvox packages available in any repo), this will also successfully replace the Perforce agent with openvox agents.